app/models/portfolio_item.rb

```diff
@@ -31,8 +31,10 @@ def metadata
   end
 
   def validate_deletable
-    unless deletable?
-      errors.add(:base, "cannot be deleted because it is used by some order processes")
+    order_process_ids = OrderProcess.where(:before_portfolio_item_id => id).or(OrderProcess.where(:after_portfolio_item_id => id)).pluck(:id)
+
+    unless order_process_ids.empty?
+      errors.add(:base, "cannot be deleted because it is used by order processes #{order_process_ids.uniq}")
```

Comment:
This message is eventually shown to the user, right? Does it make sense to show them the IDs versus the names? I don't know how useful the IDs are to someone who is viewing the content in the UI. On the other hand, for an API user, the IDs are probably really useful because they can then just hit the API endpoint with the right ID.